core/controllers/admin.py

```diff
@@ -1008,3 +1009,68 @@ def put(self):
         user_services.log_username_change(
             self.user_id, old_username, new_username)
         self.render_json({})
+
+
+class NumberOfDeletionRequestsHandler(base.BaseHandler):
+    """Handler for deleting account via admin page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        self.render_json({
+            'number_of_pending_deletion_models': (
+                wipeout_service.get_number_of_pending_deletion_requests())
+        })
+
+
+class DeleteAccountHandler(base.BaseHandler):
+    """Handler for deleting account via admin page."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        email = self.payload.get('email', None)
```

Comment:
Shouldn't deletion be done by username?

Reply:
We currently do not save the username in the pending deletion request, thus I use email here. What needs to be said that this is mostly for e2e tests.

Reply:
Done, removed.